packages/components/src/powered-by/styles.scss

```diff
@@ -0,0 +1,21 @@
+@import '@wordpress/base-styles/variables';
+
+// TODO: consider using a wrapper to ensure that the flexbox layout is applied
+// even in case of the display prop being overridden.
+.a8c-components-powered-by {
+	display: flex;
+	align-items: center;
+	flex-wrap: wrap;
+	gap: $grid-unit-10;
+
+	margin: 0;
+	padding: 0;
+	width: fit-content;
+
+	.a8c-components-powered-by__text {
+		// TODO: add any typography-related styles?
+		// (font size, font weight, line-height, , etc.)
+		// It may help with ensuring a consistent height too?
+		white-space: nowrap;
+	}
```

Comment:
@jameskoster no rush at all, but I'd like to hear more about the inheriting contextual font styles.

I assume that when `Theme` will be released (and support typography), it will be ok to specify font styles explicitly?

Reply:
The issue originally came up because there was a need to use this component in environments with different body copy sizes (14px and 13px). If the component is opinionated about its own font size, then it's going to look uncanny in whichever of those environments it's not aligned with.

> I assume that when `Theme` will be released (and support typography), it will be ok to specify font styles explicitly?

Yes.